app/assets/javascripts/subscription.js

```diff
@@ -0,0 +1,23 @@
+$("#toggler").click(function($event){
+  $event.preventDefault();
+  var link = $(this);
+
+  $.ajax({
+      type: $(this).data("type"),
+      url: $(this).data("path"),
+      success: toggle_subscription(link),
+      error: function (){
+        alert("Something went wrong.")
```

Comment:
Is this something that would come up often? If so, maybe we can create a nicer method of showing that the subscription failed?